app/src/main/java/com/duckduckgo/app/browser/BrowserChromeClient.kt

```diff
@@ -46,6 +46,11 @@ class BrowserChromeClient @Inject constructor(
 
     private var customView: View? = null
 
+    override fun onConsoleMessage(consoleMessage: ConsoleMessage): Boolean {
+        Timber.v("consoleMessage: %s", consoleMessage.message())
```

Comment:
Is this only necessary for debugging purposes?

Reply:
Right now, it's how we discover any errors in the autofill integration - we can see the errors (e.g., JSON schema violation) in the console. So it's necessary to have this logging in place, at least while we're still developing this). Longer term, it's probably something we can remove, again.